loadgen/loadgen.cc

```diff
@@ -1468,6 +1468,24 @@ void StartTest(SystemUnderTest* sut, QuerySampleLibrary* qsl,
     detail("*TestSettings (performance_sample_count_override) can override");
     detail("*Refer to Effective Settings for actual value");
   });
+
+  // Look for Audit Config file to override TestSettings during audit
+  std::ifstream audit_config("audit.config");
```

Comment:
I feel there is a need for a separate config such as audit.config which is loaded from StartTest and not by the app such as mlperf.config, especially for auditing. 

mlperf.config is loaded by the app and the app itself can override these settings in main after FromConfig is called. It is possible that the auditor will not have access to make changes to the main app to prevent overriding the settings provided by mlperf.config which could limit passing test settings for auditing. 

Also Audit.config can set settings other than those required to satisfy rules, such as random seeds or logging accuracy in performance mode.

Reply:
I learned before that my view that mlperf.conf just set the rules is too limited, so mode and co are perfectly good :)